src/central-intro.rst

```diff
@@ -17,28 +17,30 @@ Here are some of the major features we support today:
  - User accounts and management
  - Role-based user permissioning
  - Projects to organize users, permissions, and forms
- - Form and submission upload and management
+ - Form upload and management
 
    - With support for form version updates
    - With drafts and testing on initial creation, and on version updates
-   - With form and submission multimedia or data attachments
-   - With a table preview of submission data
+   - With form multimedia or data attachments
    - Encrypted forms (self-supplied or project managed keys)
+
+ - Submission upload and management
+
+   - From any compliant data collection client,
```

Comment:
I'd replace both the first two with...
- Form filling from our mobile app or a web browser

Reply:
ditto the other comment but i don't have an opinion.